Editor/Window/ConnectToFleetInput.cs

```diff
@@ -27,17 +27,21 @@ public class ConnectToFleetInput : StatefulInput
         private readonly VisualElement _container;
         private FleetStatus _fleetState;
         private List<FleetAttributes> _fleetAttributes = new List<FleetAttributes>();
-
+        private ElementLocalizer elementLocalizer;
+        private StatusBox _connectToAnywhereStatusBox;
+        
         public ConnectToFleetInput(VisualElement container, StateManager stateManager, FleetStatus initialState)
         {
             _container = container;
             _fleetState = initialState;
             _stateManager = stateManager;
+            elementLocalizer = new ElementLocalizer(_container);
```

Comment:
I don't see this referenced anywhere - could you clean up if that's the case?

Reply:
Cleaned up.